xbmcgui.py

```diff
@@ -338,8 +338,12 @@ def __str__(self):
             for key in keys:
                 value = "%s    - %s: %s\n" % (value, key, self.__properties[key])
 
+            if self.PY2:
+                return value.encode('utf-8') if type(value) == unicode else value
```

Comment:
Would `isinstance()` be better here? I think it is preferred for type testing?

Reply:
You are right. I'll change it.